DolbyIO/DolbyIO.Android.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<root xmlns:android="http://schemas.android.com/apk/res/android">
+  <init>
+    <log text="DolbyIO plugin init"/>
+  </init>
+
+  <gradleProperties>
+    <insert>
+org.gradle.jvmargs=-Xmx1536M --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
+ANDROID_BUILD_TOOLS_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
+ANDROID_TOOLS_BUILD_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
```

Comment:
What do all of these mean? Why is it 7.3.1 here and 7.4.1 in Engine?

Reply:
we can definitely change to 7.4.1, I was making some changes locally and probably made a typo in my unreal env :p so mistakenly made 7.3.1 in that configuration. Let's bump it 👍

Reply:
Setting these to 7.4.1 results in: `Minimum supported Gradle version is 7.5. Current version is 7.4.1. If using the gradle wrapper, try editing the distributionUrl in Z:\gradle\wrapper\gradle-wrapper.properties to gradle-7.5-all.zip`

Reply:
My bad, I was reading it on mobile and I thought that discussion was about the distribution url completely. No we need to use AGP 7.3.1